Kite SDK/PSPrintSDK/OLPrintJob.h

```diff
@@ -24,6 +24,9 @@
 @property (nonatomic, readonly) NSUInteger quantity;
 @property (nonatomic, readonly) NSString *templateId;
 @property (nonatomic, readonly) NSArray *currenciesSupported;
+@property (nonatomic, strong) OLAddress *address;
+@property (nonatomic, strong) NSString *uuid;
```

Comment:
I'm not convinced this is something we want in the public interface -- surely just keep it private an initialise internally? Update that then feel free to merge this.